### PR TITLE
remove trailing hyperlink underlines

### PR DESCRIPTION
**Pull request by denis-yuen**
**Description**
Replacing hyperlinked trailing spaces with non-hyperlinked non-breaking spaces

**Issue**
https://ucsc-cgl.atlassian.net/browse/DOCK-2007

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

**Comment by denis-yuen** [comment]
for Monday: husky doing some extra formatting

**Comment by svonworl** [comment]
Found more trailing spaces in the "Source Repositories" box on both the tool and container pages.

File: src/app/workflow/workflow.component.html [hunk 1]
```diff
@@ -106,7 +106,11 @@ <h3>
       <span *ngIf="(categories$ | async).length > 0" class="inline-block mr-5 mt-1 mb-1">
         <span class="size-small mr-2">Categories</span>
         <mat-chip-list class="bubble-list">
-          <app-category-button *ngFor="let category of (categories$ | async)" [category]="category" entryType="workflow"></app-category-button>
+          <app-category-button
```
Comment (svonworl) [inline src/app/workflow/workflow.component.html:109]: For the sake of knowledge, was there anything wrong with this tag, besides the extra parens?

**Review by garyluu — APPROVED** [review]

File: src/app/workflow/workflow.component.html (hunk 1 above)
Comment (denis-yuen) [reply src/app/workflow/workflow.component.html:109]: I doubt it, this is one of the automatically husky reformatings

**Comment by denis-yuen** [comment]
> Found more trailing spaces in the links in the "Source Repositories" box on the tool and container pages.

That's what I thought too at first, then I noticed that the "pop-out" icon is part of the same hyperlink. So instead of having "link space same-link", I just left it alone.

**Review by NatalieEO — APPROVED** [review]

**Pull request merged**